.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
     {
       "mode": "auto"
     }
-  ]
+  ],
+  "editor.formatOnSave": true
```

Comment:
I'm wondering why your vscode settings are in the code base. Shouldn't be this file gitignored?